### ISSUE TITLE
remove too restrictive axiom that nervous systems end during fully formed stages

### ISSUE THREAD

**Issue by cmungall** [obophenotype/uberon#3265]
Hey ontobot! apply

- delete edge UBERON:0001016 RO:0002492 UBERON:0000066

I believe this is too restrictive for holometabolous organisms, where there is no continuity between nervous system in stages, but @gouttegd @dosumis should confirm.

**Comment by gouttegd** [comment]
If I understand correctly, the rationale for the deletion is that we may have, for example, a “larval nervous system” that is classified as a “nervous system” but only exists, by definition, in larval stages, ergo it is wrong to say that a “nervous system” ceases to exist only from a fully formed stage?

If so, that looks good to me.

**Comment by gouttegd** [comment]
Unrelated to the proposed change itself: the ontobot call failed because it couldn’t find a `defusedxml` Python package. Seems like a bug in the `ontobot-change-agent` that may have forgotten to declare one of its dependencies.

**Comment by cmungall** [comment]
@gouttegd that is correct. But we do get into ship of theseus territory here. But where the ship is extensively remodeled and rewired.

Ultimately individual AOs would be the arbitrer here. If you want to put a RO:0002494 *transformation of* relationships between

FBbt:00001911 ! embryonic/larval nervous system
FBbt:00110192 ! pupal nervous system
FBbt:00003559 ! adult nervous system

then keeping the uberon axioms is justified.

If you think some other relationship is in order we should likely remove it

From the RO paper:


Transformation_of
When an embryonic oenocyte (a type of insect cell) is transformed into a larval oenocyte, one and the same continuant entity preserves its identity while instantiating distinct classes at distinct times. The class-level relation transformation_of obtains between continuant classes C and C1 wherever each instance of the class C is such as to have existed at some earlier time as an instance of the distinct class C1 (see Figure [2](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2005-6-5-r46#Fig2)). This relation is illustrated first of all at the molecular level of granularity by the relation between mature RNA and the pre-RNA from which it is processed, or between (UV-induced) thymine-dimer and thymine dinucleotide. At coarser levels of granularity it is illustrated by the transformations involved in the creation of red blood cells, for example, from reticulocyte to erythrocyte, and by processes of development, for example, from larva to pupa, or from (post-gastrular) embryo to fetus [[27](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2005-6-5-r46#ref-CR27)] or from child to adult. It is also manifest in pathological transformations, for example, of normal colon into carcinomatous colon. In each such case, one and the same continuant entity instantiates distinct classes at different times in virtue of phenotypic changes.

Figure 2
![img](https://media.springernature.com/full/springer-static/image/art%3A10.1186%2Fgb-2005-6-5-r46/MediaObjects/13059_2004_Article_1081_Fig2_HTML.jpg?as=webp)

[Full size image](https://genomebiology.biomedcentral.com/articles/10.1186/gb-2005-6-5-r46/figures/2)
As definition for this relation we offer:

C transformation_of C1 = [definition] C and C1 for all c, t, if Cct, then there is some t1 such that C1ct1, and t1 earlier t, and there is no t2 such that Cct2 and C1ct2.

That is to say, the class C is a transformation of the class C1 if and only if every instance c of C is at some earlier time an instance of C1, and there is no time at which it is an instance of both C and C1. (The final clause, which asserts that C and C1 do not share instances at a time, is inserted in order to rule out, for example, adult human transformation_of human.)

Note that C transformation_of C1 is a statement about Cs in general. It does not tell us of C1s in general that each gives rise to some C which stands to it in a transformation_of relation.

**Comment by gouttegd** [comment]
> Ultimately individual AOs would be the arbitrer here. If you want to put a RO:0002494 (“transformation of”) relationships between
>
> FBbt:00001911 ! embryonic/larval nervous system
> FBbt:00110192 ! pupal nervous system
> FBbt:00003559 ! adult nervous system
> 
> then keeping the uberon axioms is justified.

FWIW I think that would make sense, but I’d like to have @dosumis ’s opinion on that.

**Comment by dosumis** [comment]
> Ultimately individual AOs would be the arbitrer here. If you want to put a RO:0002494 transformation of relationships between
> 
> FBbt:00001911 ! embryonic/larval nervous system
> FBbt:00110192 ! pupal nervous system
> FBbt:00003559 ! adult nervous system
> 
> then keeping the uberon axioms is justified.

If the insect nervous system consists of neurons + neuroblasts then there is continuity in cellular identity.  The adult nervous system is a mix of remodelled larval neurons + neurons born from the neuroblasts (+ some of their immediate descendants that are the parents of neurons and glia) that make up the outer layer of the CNS during larval stages.  I don't think there is any continuity of PNS though.

**Comment by Clare72** [comment]
>I don't think there is any continuity of PNS though.

If C transformation_of some C1, is it the case that every part_of C must be a transformation_of some part_of C1?
